lib/src/merge.rs

```diff
@@ -747,6 +784,183 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_get_simplified_mapping() {
+        // 1-way merge
+        assert_eq!(c(&[], &[0]).get_simplified_mapping(), vec![0]);
```

Comment:
Minor, optional: I feel like this would be clearer if you just listed the adds and removes in one list instead of using `c(...)`. As is, it's hard to compare the RHS of the assert to the LHS.

OTOH, this whole test seems less important than the others, so if this is too much of a chore, it's probably OK.

Reply:
This is for all the `test_get_simplified_mapping` tests, since it's currently hard to match up the indices inside the vector on the RHS of `assert_eq!` and the indices inside the conflicts on the LHS of the assertion.

Other tests benefit from using `c(...)`, since separating removes and adds makes things clearer there.

OTOH, I'm not sure `test_get_simplified_mapping` is even needed or has to be this long if `test_update_from_simplified` covers all the important cases.

This is not something I feel strongly about.